jerry-core/parser/js/js-lexer.c

```diff
@@ -1343,7 +1342,6 @@ lexer_check_left_paren (parser_context_t *context_p) /**< context */
   return (context_p->source_p < context_p->source_end_p
           && context_p->source_p[0] == (uint8_t) LIT_CHAR_LEFT_PAREN);
 } /* lexer_check_left_paren */
-#endif /* !CONFIG_DISABLE_ES2015_CLASS */
```

Comment:
Why do you remove this check? Isn't this an ES6 feature?

Reply:
It came from the ES6 classes pull request but I also use it for checking if we have a method (it checks to see if there is a left parenthesis after the identifier). If I keep the ifdef around it then if the user disables classes I won't have this function available.

Reply:
I mean this shorthand notation isn/t an es6 feature? Then we might need to change the ifdef.

Reply:
Done. I'm planning on adding computed properties at some point, so all these new language features to object initializers could fit nicely into this macro.